airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -142,6 +150,12 @@ class KubernetesPodOperator(BaseOperator):
     :param priority_class_name: priority class name for the launched Pod
     :param termination_grace_period: Termination grace period if task killed in UI,
         defaults to kubernetes default
+    :param pod_runtime_info_envs: (Optional) A list of environment variables,
+        to be set in the container. Cannot be updated.
+    :param configmaps: (Optional) A list of names of config maps which coselects a ConfigMap
```

Comment:
coselects -> collects ?

Reply:
It was a typo from the conversion function documentation: [V1ConfigMapEnvSource](https://github.com/kubernetes-client/python/blob/b313b5e74f7cf222ccc39d8c5bf8a07502bd6db3/kubernetes/docs/V1ConfigMapEnvSource.md) which is called for every config map in the list. Using `collects` seems more appropriate. Will do the change, thank you.